deployer/README.md

```diff
@@ -486,6 +486,9 @@ parameters added:
 * `ES_COPY_PASSWORD`, `OPS_COPY_PASSWORD` - password to use to authenticate to
   elasticsearch using username/password auth
 
+*Note:* Sending logs directly to an AWS Elasticsearch instance is not supported.  Please
+use [secure forward](#secure-forward) to understand how this may be accomplished.
```

Comment:
Does secure forward actually explain how this can be accomplished? I think we should say something along the lines of "Please use [.....) in conjunction with a Fluentd instance with the <such and such> plugin configured"